src/common/settings/login/secondfactor/SecondFactorEditModel.ts

```diff
@@ -209,7 +208,11 @@ export class SecondFactorEditModel {
 				sf.otpSecret = this.totpKeys.key
 			}
 		}
-		await this.entityClient.setup(assertNotNull(this.user.auth).secondFactors, sf)
+		let extraHeaders
```

Comment:
tiny nitpick but I'd like to give `extraHeaders` some value (`undefined`) in either `else` or by default. It looks like TDZ to me even though it's not